mindmaps-graql/src/main/java/io/mindmaps/graql/internal/reasoner/atom/ValuePredicate.java

```diff
@@ -71,7 +71,10 @@ public int equivalenceHashCode() {
     public boolean isValuePredicate(){ return true;}
 
     @Override
-    public String getPredicateValue() { return predicate.getPredicate().getValue().toString();}
+    public String getPredicateValue() {
+        if (predicate.getPredicate().getValue() == null) return "";
```

Comment:
Is it really necessary to return an empty string ? This feels like a bit of a patch in the wrong place.